README.md

```diff
@@ -89,6 +89,7 @@ When you run `bazel run ///helloworld:mynamespace.apply`, it applies this file i
 | ***name_prefix***         | `None`         | Adds prefix to the names of all resources defined in manifests.
 | ***name_suffix***         | `None`         | Adds suffix to the names of all resources defined in manifests.
 | ***patches***             | `None`         | A list of patch files to overlay the base manifests. See [Base Manifests and Overlays](#base-manifests-and-overlays).
+| ***crds***                | `None`         | A list of custom resource definitions (CRDs) files. The presence of this field is to allow kustomize be aware of CRDs and apply proper transformation for any objects in those types.
```

Comment:
Please link https://kubectl.docs.kubernetes.io/references/kustomize/kustomization/crds/ page in the parameter description.